lib/modules/datasource/docker/index.ts

```diff
@@ -714,7 +714,7 @@ export class DockerDatasource extends Datasource {
         headers,
         noAuth: true,
       });
-      labels = JSON.parse(configResponse.body).config.Labels;
+      labels = JSON.parse(configResponse.body).config?.Labels;
```

Comment:
we should log a debug message when config is missing and log the object to see why it's missing